ztp/source-crs/StorageLocalVolume.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: local.storage.openshift.io/v1
+kind: LocalVolume
+metadata:
+  name: ls-fs
+  namespace: openshift-local-storage
+spec:
+  logLevel: Normal
+  managementState: Managed
+  storageClassDevices:
+    - devicePaths:
+      - /dev/nvme1n1p1
+      - /dev/nvme1n1p2
+      - /dev/nvme1n1p3
+      - /dev/nvme1n1p4
```

Comment:
These device paths seem very site-specific. The LocalStorage object should probably not be part of the source-crs but an overlay added in a specific site PolicyGenTemplate.